app-crypt/efitools/efitools-1.8.1.ebuild

```diff
@@ -29,6 +29,11 @@ PATCHES=( "${FILESDIR}/1.7.0-Make.rules.patch"
 	"${FILESDIR}/${P}-libressl-compatibility.patch" )
 
 src_prepare() {
+	if use static; then
+		append-ldflags -static
+		sed -i '/\$(CC)/s/$/ -ldl -lz/' Makefile || die
```

Comment:
why does it need this ?  because of the -lcrypto ?

that code in the Makefile should be changed to use pkg-config, and once it does, we'd set pkg-config to pass in --static, and then the -ldl/-lz/etc... would get added automatically.

Reply:
I've just updated the branch. I changed the "sed" command to use "pkg-config".

It would be ideal if we could ask the upsteram efitools developers to change their Makefile. Meanwhile, I guess we can patch the Makefile like this.